warehouse/postgres/postgresQueryBuilder.go

```diff
@@ -0,0 +1,62 @@
+package postgres
+
+import (
+	"database/sql"
+	"fmt"
+	"strings"
+)
+
+type QueryExecution struct {
+	txn                 *sql.Tx
+	db                  *sql.DB
+	query               string
+	enableWithQueryPlan bool
+}
+
+// handleQueryExecution
+// Print execution plan if enableWithQueryPlan is set to true else return result set.
+// Currently, these statements are supported by EXPLAIN ANALYZE
+// Any SELECT, INSERT, UPDATE, DELETE, VALUES, EXECUTE, DECLARE, CREATE TABLE AS, or CREATE MATERIALIZED VIEW AS statement, whose execution plan you wish to see.
```

Comment:
SELECT clause can't be sent in right ? The sql.Result interface doesn't contain rows. Should we remove this from comment then ?

Reply:
Yes.